pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -143,9 +131,10 @@ def remove_tmp_mountpoint(db, uuid):
 
 
 def remove_all_tmp_mountpoints():
-    with mount_database() as db:
-        for mp in set(tmp_mountpoints):
-            remove_tmp_mountpoint(db, mp)
+    if len(tmp_mountpoints) > 0:
```

Comment:
So is all of that tmp_mountpoints machinery actually still being used now? It feels like if it is, then we still have potential data leaks and busy mounts.

Reply:
In Anaconda mode, yes. But I _think_ we can use private mounts also there -> watch for a PR.